### PR TITLE
Rename method args for consistency and better readability.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Chronos.php [hunk 1]
```diff
@@ -88,18 +88,20 @@ class Chronos extends DateTimeImmutable implements ChronosInterface
...
      *
      * @param \DateTimeInterface|string|int|null $time Fixed or relative time
-     * @param \DateTimeZone|string|null $tz The timezone for the instance
+     * @param \DateTimeZone|string|null $timezone The timezone for the instance
      */
-    public function __construct(DateTimeInterface|string|int|null $time = 'now', DateTimeZone|string|null $tz = null)
-    {
+    public function __construct(
+        DateTimeInterface|string|int|null $time = 'now',
+        DateTimeZone|string|null $timezone = null
```
Comment (othercorey) [inline src/Chronos.php:95]: Since time zone is 2 words, I think we should follow php use `timeZone` to be accurate.

Comment (othercorey) [reply src/Chronos.php:95]: Hmm, php names the class TimeZone but the constructor parameter for DateTime is all lower case `timezone`? Unfortunate.

We won't be extending DateTime though so ...

Comment (ADmad) [reply src/Chronos.php:95]: Yes, PHP capitalizes the `z` only for the class name. For arguments (`$timezone`) and methods names (`setTimezone()`) it's lowercased. I would prefer we keep the same even if we are not going to extend `DateTimeImmutable`.

Comment (othercorey) [reply src/Chronos.php:95]: yea, let's follow it.

**Review by othercorey — APPROVED** [review]

**Pull request merged**